.pre-commit-config.yaml

```diff
@@ -61,6 +61,18 @@ repos:
     args:
     - --strict
 
+- repo: https://gitlab.com/pycqa/flake8.git
+  rev: 3.8.3
+  hooks:
+  - id: flake8
+    language_version: python3
+    additional_dependencies:
+    - flake8-2020>=1.6.0
+    - flake8-docstrings
+    - flake8-pytest-style>=1.0.0
+    - wemake-python-styleguide
+    types: [file]
```

Comment:
this is probably unnecessary + it likely causes pre-commit to directly pass random files to flake8
```suggestion
```

Reply:
Yeah, it is. Thx.